README.md

```diff
@@ -237,24 +267,34 @@ class BiometricPopup extends Component {
     return Platform.Version < 23;
   }
 
-  authCurrent() {
+  _androidTouchID() {
+    FingerprintScanner.release()
     FingerprintScanner
-      .authenticate({ description: 'Log in with Biometrics' })
+      .authenticate({
+        description: 'Scan your fingerprint on the device scanner to continue',
+        cancelButton: 'cancel',
```

Comment:
Was the android implementation using a different signature than iOS before? It is inconvenient to have separate APIs for the platforms, but maybe that is unavoidable